exes/mqtt-broadcaster/inc/mqtt_client.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <openssl/ssl.h>
+#include <async_mqtt/buffer.hpp>
+#include <async_mqtt/endpoint.hpp>
+#include <async_mqtt/packet/qos.hpp>
+#include <async_mqtt/packet/v5_publish.hpp>
+#include <async_mqtt/predefined_underlying_layer.hpp>
+#include <async_mqtt/tls.hpp>
+#include <boost/asio/any_io_executor.hpp>
+#include <boost/asio/awaitable.hpp>
```

Comment:
can be forward declared

Reply:
Changing to just 
```cpp
#include <boost/asio.hpp>
```

I was under the impression that more specific includes would save time when compiling but I was wrong so the specific includes are unnecessary.

Reply:
Newest version attempts to forward declare as much as possible.